cmd/migrate/renames.go

```diff
@@ -42,12 +42,17 @@ func applyRenames(block *hclwrite.Block) {
 func renameResourceReferences(body *hclwrite.Body) {
 	// Map of old resource names to new ones
 	resourceTypeRenames := map[string]string{
-		"cloudflare_access_policy": "cloudflare_zero_trust_access_policy",
+		"cloudflare_access_policy":      "cloudflare_zero_trust_access_policy",
 		"cloudflare_access_application": "cloudflare_zero_trust_access_application",
-		"cloudflare_access_group": "cloudflare_zero_trust_access_group",
-		// Add more renames as needed
+		"cloudflare_access_group":       "cloudflare_zero_trust_access_group",
+		// Workers resource renames
+		"cloudflare_worker_route":        "cloudflare_workers_route",
+		"cloudflare_worker_script":       "cloudflare_workers_script",
+		"cloudflare_worker_cron_trigger": "cloudflare_workers_cron_trigger",
+		"cloudflare_worker_domain":       "cloudflare_workers_custom_domain",
+		"cloudflare_worker_secret":       "cloudflare_workers_secret",
```

Comment:
These two aren't actually the same resource. In v5, `cloudflare_workers_secret` hits the **Workers for Platforms** APIs (and as a result never worked as users expected, so we dropped it from v5.3.0). Existing `cloudflare_worker_secret` resources should be migrated to a binding of `secret_text` type on the corresponding `cloudflare_workers_script` resource.